src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -662,6 +662,36 @@ internal static void SetCurrentThreadUiCulture(CultureInfo uiCultureInfo)
         #region Misc
 
         /// <summary>
+        /// Facade for Convert.ToBase64String(bytes, Base64FormattingOptions.InsertLineBreaks)
+        /// Inserts line breaks after every 76 characters in the string representation.
+        /// </summary>
+        internal static string ToBase64StringWithLineBreaks(byte[] bytes)
+        {
+#if CORECLR
+            // Inserts line breaks after every 76 characters in the string representation.
+            string encodedRawString = Convert.ToBase64String(bytes);
+            if (encodedRawString.Length <= 76)
+                return encodedRawString;
+            
+            StringBuilder builder = new StringBuilder(encodedRawString.Length);
+            int index = 0, remainingLen = encodedRawString.Length;
+            while (remainingLen > 76)
```

Comment:
@daxian-dbw Perhaps it makes sense to open new Issue to migrate to .Net Core method when it become available?